effects/cats/src/test/scalajvm/sttp/client3/impl/cats/HttpClientCatsHttpTest.scala

```diff
@@ -0,0 +1,10 @@
+package sttp.client3.impl.cats
+
+import cats.effect.IO
+import sttp.client3.testing.HttpTest
+
+class HttpClientCatsHttpTest extends HttpTest[IO] with HttpClientCatsTestBase {
+  override def supportsHostHeaderOverride = false
+
+  override def supportsDeflateWrapperChecking = false
```

Comment:
it seems that's `true` for `HttpClientZioHttpTest`, maybe we can flip it here as well?

Reply:
Zio have somewhat different impl of streaming this one is based on HttpClientFuture which also have this flags set to false